content/cn/blog/_posts/2018-05-30-say-hello-to-discuss-kubernetes.md

```diff
@@ -0,0 +1,57 @@
+---
+title: ' Kubernetes 1.11：让我们开始讨论Kubernetes '
+cn-approvers:
+- congfairy
+layout: blog
+date: 2018-05-30
+---
+
+<!--   
+
+Author: Jorge Castro (Heptio)
+
+-->
+
+作者: Jorge Castro (Heptio)
+
+<!-- 
+
+Communication is key when it comes to engaging a community of over 35,000 people in a global and remote environment. Keeping track of everything in the Kubernetes community can be an overwhelming task. On one hand we have our official resources, like Stack Overflow, GitHub, and the mailing lists, and on the other we have more ephemeral resources like Slack, where you can hop in, chat with someone, and then go on your merry way. 
+
+-->
+
+成为在全球范围有超过35,000人的社区时沟通是非常关键的。 跟踪Kubernetes社区中的所有内容可能是一项艰巨的任务。 一方面，我们有官方资源，如Stack Overflow，GitHub和邮件列表，另一方面，我们有更多实时的资源，如Slack，你可以在那里跳，与某人聊天，然后继续按你喜欢的方式去做。
+
+<!--
+
+Slack is great for casual and timely conversations and keeping up with other community members, but communication can't be easily referenced in the future. Plus it can be hard to raise your hand in a room filled with 35,000 participants and find a voice. Mailing lists are useful when trying to reach a specific group of people with a particular ask and want to keep track of responses on the thread, but can be daunting with a large amount of people. Stack Overflow and GitHub are ideal for collaborating on projects or questions that involve code and need to be searchable in the future, but certain topics like "What's your favorite CI/CD tool" or "Kubectl tips and tricks" are offtopic there.
+
+While our current assortment of communication channels are valuable in their own rights, we found that there was still a gap between email and real time chat. Across the rest of the web, many other open source projects like Docker, Mozilla, Swift, Ghost, and Chef have had success building communities on top of Discourse, an open source discussion platform. So what if we could use this tool to bring our discussions together under a modern roof, with an open API, and perhaps not let so much of our information fade into the ether? There's only one way to find out: Welcome to discuss.kubernetes.io
+
+-->
+
+Slack非常适合随意和及时的对话，并与其他社区成员保持联系，但未来很难轻易引用通信。此外，在35,000名参与者中提问并得到回答很难。邮件列表在有问题尝试联系特定人群并且想要跟踪大家的回应时非常有用，但是对于大量人员来说可能是麻烦的。 Stack Overflow和GitHub非常适合在涉及代码的项目或问题上进行协作，并且如果在将来要进行搜索也很有用，但某些主题如“你最喜欢的CI / CD工具是什么”或“[Kubectl提示和技巧]”（https：// discuss.kubernetes.io/t/kubectl-tips-and-tricks/192）“在那里是没有意义的。
```

Comment:
https：// discuss.kubernetes.io/t/kubectl-tips-and-tricks/192
to
http://discuss.kubernetes.io/t/kubectl-tips-and-tricks/192